docs/administrators/updating_the_orchestrator.rst

```diff
@@ -0,0 +1,67 @@
+.. _updating_the_orchestrator:
+
+
+Updating the orchestrator
+-------------------------
+
+Migrating from one running orchestrator to another running orchestrator
+#######################################################################
+
+This document describes the procedure to migrate from one running orchestrator
+to another one. This procedure should be followed when an in-place update of
+the orchestrator is not possible e.g. when the operating system needs to be
+updated alongside the orchestrator or when updating the orchestrator to a
+different major version.
+
+Context
++++++++
+
+The procedure below describes how to migrate from one running orchestrator
+denoted as the 'old orchestrator' to another one denoted as the 'new orchestrator'.
+
+Procedure
++++++++++
```

Comment:
At the moment we only support major upgrades from the latest version of major X to the next major X+1. I would add a per-requisite saying that:

* When the goal is to do an upgrade to a new major version, an update should be done first.
* Upgrades should be done one major at a time. So if you want to upgrade from major version X to major version X+2, you should do an upgrade from X to X+1 and then from X+1 to X+2.